lib/integer.gi

```diff
@@ -1195,7 +1195,11 @@ InstallGlobalFunction(PowerModInt,function ( r, e, m )
     # return the power
     return pow;
 end);
-
+if IsBound(POWERMODINT) then
```

Comment:
The original reasons I kept them are (a) testing, and (b) making it possible to quickly switch between branches without always having to recompile (as warnings would be shown about undeclared functions). Both were mostly useful for development of this PR. I left them in because that way it was clearly visible to everybody that I did indeed verify the new against the old implementation.

I have no problem either removing those, or else moving them to the .tst files, as you suggest.